src/main/java/io/github/oliviercailloux/y2018/apartments/utils/JsonConvert.java

```diff
@@ -34,48 +34,22 @@
 	/**
 	 * The default path to JSON file.
 	 */
-	public static final Path APARTMENT_PATH_JSON = Path.of("Apartment_Json.json");
+	public static final Path APARTMENTS_PATH_JSON = Path.of("exportApartment" + System.currentTimeMillis() + ".json");
```

Comment:
Pour tout le lancement du programme, le path ne sera calculé qu'une seule fois. Ici, une fonction permettant de calculer le path permettrait d'utiliser un path (et donc un fichier) différents  chaque fois que l'utilisateur devra générer un nouveau fichier json avec des appartements.
De plus, il serait mieux que la fonction soit en accès privé car il n'aura pas besoin de ce format de path (avec 'exportApartment"+XXX) pour mettre autre chose que des apartments.

Reply:
Fix in commit a0c4e1146c098a341303f54d0d77970254aefdb8